config/openhpc-storage-A.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # This name is used for the Heat stack and as a prefix for the
 # cluster node hostnames.
-cluster_name: openhpc-nvme
+cluster_name: openhpc-storage-A
```

Comment:
Is the openhpc part actually relevant when on storage nodes?

Reply:
I don't believe they use any components from OpenHPC repos.